methods/transient/Transient_base.ipynb

```diff
@@ -46,19 +46,25 @@
     "NISAR must be able to constrain displacements in two look directions (left-looking on both ascending and descending tracks) for the active area of 70% of target sites with an accuracy that scales with baseline distance L between any two locations within a scene. This pertains to all directly measured or inferred 12-day interferograms over the duration of the mission. Here, accuracy is calculated using L in kilometers but with the units removed, at or better than 100 m resolution. The 12-day time scale corresponds to a repeat NISAR pass on each of the ascending and descending satellite tracks, from which interferograms can be made and displacements estimated. The NISAR mission has compiled a list of 2000 global targets covering areas of known or potential transient deformation related to the processes specified in the requirement (NISAR Handbook Appendix H, 2018). These targets include all Earth's active volcanoes above sea-level, areas of rapid glacial mass changes, selected deforming reservoirs of water, oil, gas, CO2 and steam and landslide-prone areas near major population centers, as well as sites where selected disaster-related events have occurred. \n",
```

Comment:
<p>@EJFielding is this ok or did we say less than 3 years?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>The Secular and Coseismic requirements mention a three-year interval, but Transient does not. If Sentinel-1 acquisitions are complete, then we will get 30 cycles per year. With the sampling scheme of the Transient notebook to only use pairs without common dates, then that would leave only 15 pairs per year. Seems like we need at least two years to get a reasonable sample for the statistics. Three years is good, too. It should not be very large if we only download the 12-day pairs for the three years.</p><p> </p>

Reply:
<p>I do the change as it is Eric's requirement. Eric says:</p><pre class="ql-syntax" spellcheck="false">but the NISAR primary mission is still defined as 3 years, so we will need to validate NISAR data before 3 years are passed. I think we should use 3 years or less for the notebooks.

&nbsp;

I used only one year for the Coseismic requirement notebook.
</pre>

Reply:
<p>I believe that the track 144 over the Central Valley has both S1A and S1B acquisitions, so that means that we can get 30 independent pairs with 12-day lengths (15 S1A-S1A pairs and 15 S1B-S1B pairs). Is 30 independent pairs enough for the Transient statistics?</p>

Reply:
I only get 34 products downloaded and 18 independent pairs are left. I don't know why this is less than your estimation. 18 products may not be enough.